### PR TITLE
Adding exchanges to trades

### LINKED ISSUE #693: Add Maker Relayer Address and Taker Relayer Address
In trades table, we need to add Maker Relayer Address and Taker Relayer Address. It will help to filter data easier

### PR DESCRIPTION
**Pull request by thanhnguyennguyen**
Changes in this PR:
- [x] Remove dependencies from https://github.com/tomochain/tomox-sdk
- [x] Lowercase addresses, hashes for easily filtering in Tomoscan
- [x] Adding makerExchange, takerExchange to trades. Fix #693

### CODE REVIEW

File: tomox/tomox.go [hunk 1]
```diff
@@ -1236,7 +1235,7 @@ func (tomox *TomoX) SyncDataToSDKNode(txDataMatch TxDataMatch, txHash common.Has
 		tradeSDK.PairName = order.PairName
 		tradeSDK.BaseToken = order.BaseToken
 		tradeSDK.QuoteToken = order.QuoteToken
-		tradeSDK.Status = sdktypes.TradeStatusSuccess
+		tradeSDK.Status = TradeStatusSuccess
```
Comment (thanhnguyennguyen) [inline tomox/tomox.go:1257]: update makerExchange, takerExchange to trades here

File: tomox/trade.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
+package tomox
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto/sha3"
+	"github.com/globalsign/mgo/bson"
+	"github.com/tomochain/tomox-sdk/utils/math"
```
Comment (thanhnguyennguyen) [inline tomox/trade.go]: remove this dependency from tomox-sdk

Comment (thanhnguyennguyen) [reply tomox/trade.go]: here as well : https://github.com/tomochain/tomochain/pull/695/files#diff-177d273c43f8fc4c44c67dbfdefa9a03R11

Comment (thanhnguyennguyen) [reply tomox/trade.go]: updated

File: tomox/tomox.go (hunk 1 above)
Comment (thanhnguyennguyen) [reply tomox/tomox.go:1257]: added

**Comment by thanhnguyennguyen** [comment]
tested! It works on my machine :))


image

**Review by thanhson1085 — APPROVED** [review]

**Pull request merged**